packages/state-manager/src/sagas/files/files.master.saga.ts

```diff
@@ -17,6 +17,11 @@ export function* filesMasterSaga(socket: Socket): Generator {
       networkActions.addInitializedCommunity.type,
       resetTransferSpeedSaga,
     ),
+    takeEvery(
+      networkActions.addInitializedCommunity.type,
```

Comment:
Below you have identical observer on checkForMissingFiles action. Why don't you put this action in the right place in saga? I think it'd be more intuitive,  better readable and easier to test